test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,25 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
+    it('returs a dictionary on or the specific error', function()
+        -- Returns the dictionary on success otherwise a specified error
+        -- if the resources were not present on the system.
+
+        local status, rv = pcall(meths.get_keyboard_mods_state)
+        if status then
+          for k, v in pairs(rv) do
+            eq(true, ({CapsLock=true, NumLock=true, ScrollLock=true})[k])
+            eq('boolean', type(v))
+          end
+        else
+          if os_name() == 'unix' then
+            eq("Unable to load ", string.match(rv, "Unable to load *"))
+          elseif os_name() == 'osx' then
+            eq("Unable to get service ",
+                 string.match(rv, "Unable to get service *"))
```

Comment:
This should be aligned with first `"` on the previous line.

Reply:
2 extra spaces, OK